packages/sanity/src/desk/panes/document/documentInspector/Resizer.tsx

```diff
@@ -0,0 +1,89 @@
+import React, {useCallback, useRef} from 'react'
+import styled from 'styled-components'
+
+const Root = styled.div`
+  position: absolute;
+  top: 0;
+  left: -4px;
+  bottom: 0;
+  width: 9px;
+  z-index: 201;
+  cursor: col-resize;
```

Comment:
The resize UI looks different between panes and inspectors and we use different cursors (`col-resize` here and `ew-resize` in panes) – should we unify this?

<img width="695" alt="Screenshot 2023-05-29 at 14 28 16" src="https://github.com/sanity-io/sanity/assets/15094168/2387f451-7f86-40ef-a1fa-eebc60b9b684">

Reply:
You're totally right – will align on with the pane resizer 👍